fabric-biomes-v1/build.gradle

```diff
@@ -0,0 +1,7 @@
+archivesBaseName = "fabric-biomes-v1"
+version = getSubprojectVersion(project, "1.0.0")
+
+dependencies {
+	compile project(path: ':fabric-biomes-v1', configuration: 'dev')
+	testmodCompile project(path: ':fabric-content-registries-v1', configuration: 'dev')
```

Comment:
so what's this for?

Reply:
didn't you want to do stuff with syncing later?

Reply:
ah yes but shouldn't the scope be `compile` and not `testmodCompile`?

Reply:
I just copied code from armour module idk I never use gradle modules

Reply:
the armor module added an item so it needed it

Reply:
explain